vdyp-sindex/src/main/Sindxdll.java

```diff
@@ -0,0 +1,3748 @@
+  SI_SPEC_CWI, /* SI_CWI_NIGH */
+  SI_SPEC_BP, /* SI_BP_CURTIS */
+  SI_SPEC_HWC, /* SI_HWC_NIGHGI99 */
+  SI_SPEC_SS, /* SI_SS_NIGHGI99 */
+  SI_SPEC_SW, /* SI_SW_NIGHGI99 */
+  SI_SPEC_LW, /* SI_LW_NIGHGI */
+  SI_SPEC_SW, /* SI_SW_NIGHTA */
+  SI_SPEC_CWI, /* SI_CWI_NIGHGI */
+  SI_SPEC_SW, /* SI_SW_GOUDNIGH */
+  SI_SPEC_HM, /* SI_HM_MEANS */
+  SI_SPEC_SE, /* SI_SE_CHEN */
+  SI_SPEC_FDC, /* SI_FDC_NIGHTA */
+  SI_SPEC_FDC, /* SI_FDC_BRUCENIGH */
+  SI_SPEC_LW, /* SI_LW_NIGH */
+  SI_SPEC_SB, /* SI_SB_NIGH */
+  SI_SPEC_AT, /* SI_AT_NIGH */
+  SI_SPEC_BL, /* SI_BL_CHENAC */
+  SI_SPEC_BP, /* SI_BP_CURTISAC */
+  SI_SPEC_HM, /* SI_HM_MEANSAC */
+  SI_SPEC_FDI, /* SI_FDI_THROWERAC */
+  SI_SPEC_ACB, /* SI_ACB_HUANGAC */
+  SI_SPEC_PW, /* SI_PW_CURTISAC */
+  SI_SPEC_HWC, /* SI_HWC_WILEYAC */
+  SI_SPEC_FDC, /* SI_FDC_BRUCEAC */
+  SI_SPEC_CWC, /* SI_CWC_KURUCZAC */
+  SI_SPEC_BA, /* SI_BA_KURUCZ82AC */
+  SI_SPEC_ACT, /* SI_ACT_THROWERAC */
+  SI_SPEC_PY, /* SI_PY_HANNAC */
+  SI_SPEC_SE, /* SI_SE_CHENAC */
+  SI_SPEC_SW, /* SI_SW_GOUDIE_NATAC */
+  SI_SPEC_PY, /* SI_PY_NIGH */
+  SI_SPEC_PY, /* SI_PY_NIGHGI */
+  SI_SPEC_PLI, /* SI_PLI_NIGHTA2004 */
+  SI_SPEC_SE,  /* SI_NIGHTA */
+  SI_SPEC_SW,  /* SI_NIGHTA2004 */
+  SI_SPEC_SW,  /* SI_GOUDIE_PLAAC */
+  SI_SPEC_PJ,  /* SI_HUANG */
+  SI_SPEC_PJ,  /* SI_HUANGAC */
+  SI_SPEC_SW,  /* SI_SW_NIGHGI2004 */
+  SI_SPEC_EP,  /* SI_EP_NIGH */
+  SI_SPEC_BA,  /* SI_BA_NIGHGI */
+  SI_SPEC_BA,  /* SI_BA_NIGH */
+  SI_SPEC_SW,  /* SI_SW_HU_GARCIA */
+  SI_SPEC_SE,  /* SI_SE_NIGHGI */
+  SI_SPEC_SE,  /* SI_SE_NIGH */
+  SI_SPEC_CWC  /* SI_CWC_NIGH */
+};
+
+//BOOL APIENTRY DllMain function removed
+
+//Removed WINAPI from all subsequent functions
+
+/**
+ * Sindex_VersionNumber
+ * ===================
+ * Returns the version number of the Sindex routines.
+ *
+ * @param None
+ * @return int The number indicating the version of the Sindex routines.
+ *
+ * @remarks The format of the number is always in the form of:
+ *           Mmm
+ *           where M: the major release number (1, 2, ...)
+ *                 mm: the minor release number (0, 1, ..., 99)
+ *
+ *           An example would be: 631, meaning version 6.31
+ *
+ *          If the major release is greater than what your application expects,
+ *          assume that the Sindex routines cannot be used, and that the user
+ *          needs to obtain a newer version of sindex.dll.
+ *
+ *          Minor release changes may include:
+ *          - Addition of a function.
+ *          - Changed return values (e.g., error messages).
+ *          - Iterating for solutions may generate different results.
+ *          - Bug fixes in implementation of site index equations.
+ *          - Addition of species.
+ *          - Addition of curve sources (equations).
+ *          - Change of default curve for a species.
+ *          - Change of mapping species to a different species.
+ */
+public static short Sindex_VersionNumber() {

```

Comment:
Sorry, I should have noticed this on the first pass.  The `Sindex_` prefix isn't needed as this is inside the `Sindexdll` class which serves as a namespace without needing to add a prefix to the methods.